treadmill/infra/setup/node.py

```diff
@@ -0,0 +1,21 @@
+from treadmill.infra.instances import Instances
+
+
+class Node:
+    def __init__(self):
+        self.instances = None
+
+    def setup(self, Count=1, SubnetId='', SecurityGroupIds=()):
+        self.instances = Instances.create(
+            Name='TreadmillNode',
+            ImageId='ami-9e2f0988',
```

Comment:
same here - make ami constant, and pass it as parameter from above. this is top level argument.